server/serverless.yml

```diff
@@ -381,6 +424,18 @@ resources:
         Table: ${self:provider.environment.GROUPS_TABLE}
         Version: 1
 
+    ## #Odavi's
+    ContactManagementTopic:
+      Type: AWS::SNS::Topic
+      Properties:
+        DisplayName: Cloud CRM automatic notifications
+        TopicName: ${self:provider.environment.AUTOMATIC_NOTIFICATIONS_TOPIC_TITLE}
+        #!?
+        Subscription:
```

Comment:
please don't create Subscribers in a Serverless file! we will create them in AWS manually